### PR TITLE
Remove debug file paths and files

### PR DESCRIPTION
**Pull request by JWZ2018**
Remove hardcoded default file paths to debugger logs
Remove empty debugger log files

### CODE REVIEW

File: src/prepack-options.js [hunk 1]
```diff
@@ -103,10 +104,9 @@ export function getSerializerOptions({
   return result;
 }
 
-export function getDebuggerOptions({
-  debugInFilePath = "./src/debugger/.sessionlogs/adapter2engine.txt",
-  debugOutFilePath = "./src/debugger/.sessionlogs/engine2adapter.txt",
-}: PrepackOptions): DebuggerOptions {
+export function getDebuggerOptions({ debugInFilePath, debugOutFilePath }: PrepackOptions): DebuggerOptions {
+  // if the debugger is started, the in and out files need to be specified
```
Comment (NTillmann) [inline src/prepack-options.js]: I don't understand this comment in connection with the invariant below which seems to enforce that they do already have to be specified.

Comment (JWZ2018) [reply src/prepack-options.js]: This comment just explains why this invariant is here.

Comment (cblappert) [reply src/prepack-options.js]: Style-wise prefer breaking && checks into multiple invariants (clearer to see which one failed when debugging) and putting comments as invariant messages (2nd arg) if necessary. In this case the comment seems redundant with the invariant.

Comment (JWZ2018) [reply src/prepack-options.js]: Updated invariants and removed comment.

**Pull request closed without merging**